csm/cli/support_bundle/bundle_generate.py

```diff
@@ -78,7 +78,7 @@ def exc_components_cmd(commands: List, bundle_id: str, path: str,
             cmd_proc = SimpleProcess(f"{command} {bundle_id} {path}")
             output, err, return_code = cmd_proc.run()
             Log.debug(f"Command Output -> {output} {err}, {return_code}")
-            if err:
+            if return_code == -1:
```

Comment:
Is -1 only failed case. mostly other than 0 is failure

Reply:
It should be return_code != 0:

Reply:
will change

Reply:
ChangedFixed